docs/modules/sort/quick_start.md

```diff
@@ -3,64 +3,43 @@ title: Deployment
 sidebar_position: 2
 ---
 
-## Set up flink environment
-Currently inlong-sort is based on flink, before you run an inlong-sort application,
-you need to set up [flink environment](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/overview/).
+## Set up Flink Environment
+Currently InLong-Sort is based on Flink, before you run an InLong-Sort Application,
+you need to set up [Flink Environment](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/overview/).
 
-Currently, inlong-sort relys on flink-1.13.5. Chose `flink-1.13.5-bin-scala_2.11.tgz` when downloading package.
+Currently, InLong-Sort relys on Flink-1.13.5. Chose `flink-1.13.5-bin-scala_2.11.tgz` when downloading package.
 
-Once your flink environment is set up, you can visit web ui of flink, whose address is stored in `/${your_flink_path}/conf/masters`.
+Once your Flink Environment is set up, you can visit Web UI of Flink, whose address is stored in `/${your_flink_path}/conf/masters`.
 
 ## Prepare installation files
 All installation files at `inlong-sort` directory.
 
 ## Starting an inlong-sort application
-Now you can submit job to flink with the jar compiled, refer to [how to submit job to flink](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/cli/#submitting-a-job).
+Now you can submit job to Flink with the jar compiled, refer to [How to submit job to Flink](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/cli/#submitting-a-job).
 
 Example：
 ```
-./bin/flink run -c org.apache.inlong.sort.singletenant.flink.Entrance inlong-sort/sort-[version].jar \
---cluster-id debezium2hive --dataflow.info.file /YOUR_DATAFLOW_INFO_DIR/debezium-to-hive.json \
---source.type pulsar --sink.type hive --sink.hive.rolling-policy.rollover-interval 60000 \
---metrics.audit.proxy.hosts 127.0.0.1:10081 --sink.hive.rolling-policy.check-interval 30000
+./bin/flink run -c org.apache.inlong.sort.Entrance inlong-sort/sort-[version].jar \
+--group.info.file /YOUR_DATAFLOW_INFO_DIR/mysql-to-kafka.json
 ```
 
 Notice：
 
-- `-c org.apache.inlong.sort.singletenant.flink.Entrance` is the main class name
+- `-c org.apache.inlong.sort.Entrance` is the main class name
 
 - `inlong-sort/sort-[version].jar` is the compiled jar
 
 ## Necessary configurations
-- `--cluster-id ` represent a specified inlong-sort application, same as the configuration of `sort.appName` in inlong-manager
-- `--dataflow.info.file` dataflow configuration file path
-- `--source.type` source of the application, currently "pulsar" is supported
-- `--sink.type` sink of the application, currently "clickhouse", "hive", "iceberg", "kafka" are supported
-- `--metrics.audit.proxy.hosts` audit proxy host address for reporting audit metrics
-
+- `--group.info.file` dataflow configuration file path
```

Comment:
notice if we need out Extended extrace and load，we should put our connector into ${FLINK_HOME}/lib folder.For example, if we want mysql to hive，we shoule put `sort-connector-mysql-cdc.jar` and `sort-connector-hive.jar` in ${FLINK_HOME}/lib

Reply:
It is not necessary, these dependencies will be automatically imported after the sort-dist-[version].jar package is introduced